.github/dependabot.yml

```
@@ -4,4 +4,16 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "daily"
\ No newline at end of file
+      interval: "daily"
+
+  # Maintain dependencies for Go modules
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Maintain dependencies for Docker images
```

I always suggest writing all comments as sentences, then there is no arguing about capitalization and dots :smile: 
```suggestion
  # Maintain dependencies for Docker images.
```